Sources/DataCache.swift

```diff
@@ -367,12 +376,18 @@ extension DataCache {
                 print("DataCache: Error while iterating files \(error.localizedDescription)")
             }
         }
-        
+
         return (urlsToDelete, diskCacheSize, cachedFiles)
     }
-    
+
     func cachePath(forKey key: String) -> String {
-        let fileName = key.md5
+        let fileName: String
+        if #available(iOS 13, *) {
+            fileName = key.md5String
+        } else {
+            fileName =  = key.md5
```

Comment:
I think you mistyped here :)
Apart from that LGTM

Reply:
Fixed :)